README.md

```diff
@@ -4,22 +4,53 @@
 
 Authenticator is a shared component which encapsulates authentication services required by OHDSI tools. Its goal is to reduce amount of effort required to implement and support authentication in multiple applications, provide single source of truth. 
 
+Currently supported modes:
+- Standard
+- Proxy
+
 Currently supported methods:
 - Database (JDBC)
 - REST
 - Github
 - LDAP
 - Active Directory
 
-## Configuration
+
+
+## PROXY authentication modes 
+The proxy mode means that your application is deployed on an environment behind a proxy. 
+This proxy takes full responsibility of the authentication, and transfer to us only a access token in a request header.
```

Comment:
Please use Grammarly or whatever else. There are 2 language errors only on this line

Reply:
fixed